db/schema.rb

```diff
@@ -32,12 +32,12 @@
     t.string "postcode"
     t.string "ofsted_number"
     t.integer "failed_attempts", default: 0, null: false
-    t.datetime "locked_at", precision: nil
+    t.datetime "locked_at"
     t.string "unlock_token"
     t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true
     t.index ["email"], name: "index_users_on_email", unique: true
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
-    t.index ["unlock_token"], name: "index_users_on_unlock_token"
+    t.index ["unlock_token"], name: "index_users_on_unlock_token", unique: true
```

Comment:
It'll be something about your postgres vs the container. I've seen this before. Can you send me your db details so I can workout the cause, be it version or compile settings etc.

Reply:
Your database is out of synch with the code base - not necessarily my database. Especially as the last migration was by Adam not me.

TBH I keep on running `rails db:schema:load` each time I move back to main before moving to a new branch and that keeps my db in line with the current code.

I would suggest you rollback the changes to schema.rb in your local branch and then run `rails db:reset` (in the container if that is your dev environment). Then see if that causes a change to schema.

Reply:
I wasn't able to run `bin/docker-rails db:reset`, as the container was not running, but when I ran `bin/docker-dev` and then tried reset in a different terminal, it would not let me because the database was in use....

Reply:
The service running the app will need to be stopped before db:reset will work.

Reply:
@ebrett I tweaked the script to stop using grep and instead use the filter flag, but I think that might be a bad idea, remind me and I'll double check.

Reply:
Reverted `docker-rails` from a fortnight ago @ebrett

Reply:
Reverted the schema @reggieb, I still got schema changes. I will swing back around to this after the pipeline is up and be sure not to commit any unintended changes in the meantime.